src/main/resources/templates/start.html

```diff
@@ -1,10 +1,10 @@
 <!DOCTYPE html>
 <html lang="en">
-<head>
-    <meta charset="UTF-8">
-    <title>Caching In</title>
-</head>
-<body>
-Geo-caching all over the world!
-</body>
+    <head>
+        <meta charset="UTF-8">
+        <title>Caching In</title>
+    </head>
+    <body>
+        <h1>Geo-caching all over the world!</h1>
```

Comment:
Good idea.  For accessibility, and general structure, use the "h" HTML tags for organization.